Directory.Packages.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <ItemGroup>
-    <PackageVersion Include="Aspire.Hosting.Testing" Version="8.2.1" />
+    <PackageVersion Include="Aspire.Hosting.AppHost" Version="9.0.0-rc.1.24511.1" />
+    <PackageVersion Include="Aspire.Hosting.Testing" Version="9.0.0-rc.1.24511.1" />
```

Comment:
Could you add these to the dependabot config as a group so the Aspire deps get updated together?